services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -963,6 +963,18 @@ async def get_project_type(self, project_uuid: ProjectID) -> ProjectType:
                 return row[projects.c.type]
         raise ProjectNotFoundError(project_uuid=project_uuid)
 
+    async def has_permission(
+        self, user_id: UserID, project_uuid: str, permission: str
```

Comment:
MINOR: annotate permission with some Literal? Do we have something like ``PermissionsStr = Literal["read", "write", "delete"]`` etc ...??

Reply:
you can do something like so
```python
Permission = Literal["read", "write", "delete"]
```

then 
```python
 async def has_permission(
        self, user_id: UserID, project_uuid: str, permission: Permission)
```

Then the mypy, Pylance will complain if you use it wrongly